spanner/jdbc/src/main/java/com/example/spanner/jdbc/PgBatchDmlSample.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.jdbc;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Arrays;
+
+class PgBatchDmlSample {
+
+  static void pgBatchDml() throws SQLException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project";
+    String instanceId = "my-instance";
+    String databaseId = "my-database";
+    pgBatchDml(projectId, instanceId, databaseId);
+  }
+
+  static void pgBatchDml(String projectId, String instanceId, String databaseId)
+      throws SQLException {
+    try (Connection connection =
```

Comment:
Similar to [client initialization recs](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#client-initialization), a comment on the initialization of the connection object should be added.

Reply:
Done